menu.py

```diff
@@ -870,22 +618,32 @@ def previous_menu_option(self,
         target_option = list(options_enum)[target_option_idx]
         self.change_menu_option(target_option)
 
-    def change_menu_option(self,
-                           target_option: SetupOption | WikiOption | MainMenuOption | VictoryType) -> None:
+    def change_menu_option(self, target_option: MenuOptions) -> None:
         """
         Select the given menu option.
         :param target_option: The target option to be selected.
         """
         # Based on the enum type of the target option, figure out which corresponding field we need to change.
         match target_option:
-            case SetupOption():
-                self.setup_option = target_option
-            case WikiOption():
-                self.wiki_option = target_option
-            case MainMenuOption():
-                self.main_menu_option = target_option
-            case VictoryType():
-                self.victory_type = target_option
+            case SetupOption(): self.setup_option = target_option
+            case WikiOption(): self.wiki_option = target_option
+            case MainMenuOption(): self.main_menu_option = target_option
+            case VictoryType(): self.victory_type = target_option
+
+    def get_option_colour(self, option: MenuOptions) -> pyxel.Colors:
+        """
+        Determine which colour to use for drawing menu options. RED if the option is currently selected by the user,
+        and WHITE otherwise.
+        :param option: the menu option to pick the colour for.
+        :return: The appropriate colour.
+        """
+        match option:
+            case SetupOption(): field_to_check = self.setup_option
+            case WikiOption(): field_to_check = self.wiki_option
+            case MainMenuOption(): field_to_check = self.main_menu_option
+            case VictoryType(): field_to_check = self.victory_type
+
+        return pyxel.COLOR_RED if field_to_check is option else pyxel.COLOR_WHITE
```

Comment:
Pycharm is complaining that `field_to_check` might be undefined, which is true. Maybe we could just make the last case the default one?

Reply:
Won't be a problem if you change IDEs 🧠